### ISSUE TITLE
DSSS decoder is not functional

### ISSUE THREAD

**Issue by kantooon** [ericdegroot/gr-dsss#3]
The original code of the despreader was completely non-functional because the internal samples per symbol value was never set. #2 pull request fixes this issue, removes existing example and adds a more relevant BPSK DSSS example.

**Comment by ericdegroot** [comment]
Hi Adrian,

Thank you for taking to the time to make a pull request. I wrote that 
code for a grad school class a bunch of years ago. I don't know that I'm 
really in a position to evaluate your changes any more. I haven't used 
GNU Radio in a long time. We use different DSP software within the 
industry I ended up in.

May I suggest forking the project with these changes applied and making 
it available under your own name? I would be happy to place a note at 
the top of my README for folks to check yours out.

Thanks,

Eric
